extensions-contrib/materialized-view-maintenance/src/main/java/org/apache/druid/indexing/materializedview/MaterializedViewSupervisor.java

```diff
@@ -277,6 +277,12 @@ public void reset(DataSourceMetadata dataSourceMetadata)
     }
   }
 
+  @Override
+  public void resetOffsets(DataSourceMetadata resetDataSourceMetadata, boolean setOffsetsInMetadata)
```

Comment:
This method seems to be called with`setOffsetsInMetadata` always set to true

Reply:
yeah, thanks, cleaned up. This was a leftover from some refactoring. `ResetOffsets` always sets it to true and only `ResetNotice()` and `resetInternal()` have the additional parameter.